### ISSUE TITLE
топики для мультиварки RMC M92S

### ISSUE THREAD

**Issue by Svet98** [alutov/ESP32-R4sGate-for-Redmond#217]
Андрей, доброго времени суток!)
Помню на первой странице был сабж, сейчас не могу найти.
Возникли затруднения к прикручиванию к MajorDomo через MQTT, не поможите?
Спасибо)

**Comment by alutov** [comment]
Да, убрал мабж при переводе, дкмал, что инфа уже не пригодится. Но [в истории это найти можно)](https://github.com/alutov/ESP32-R4sGate-for-Redmond/tree/18e7beb6afb53c3f3cb9ebf93d8b16e6fbed320d)

**Comment by Svet98** [comment]
Спасибо!)
